changelogs/current.yaml

```diff
@@ -306,6 +306,10 @@ new_features:
     added support to allowlist headers included in the check request to gRPC authorization server (previously only available for HTTP authorization server).
     Pre-existing field :ref:`allowed_headers <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.AuthorizationRequest.allowed_headers>` is deprecated in favour
     of the new field :ref:`allowed_headers <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.allowed_headers>`.
+- area: ext_authz
+  change: |
+    added a new field :ref:`propagate_response_on_failure <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.HttpService.propagate_response_on_failure>` to control whether or
```

Comment:
May be the label should be `envoy_v3_api_field_extensions.filters.http.ext_authz.v3.HttpService.propagate_response_on_failure` to fix the docs building error?

Reply:
Thank You